pkg/cli/cmds/server.go

```diff
@@ -252,6 +253,11 @@ var ServerFlags = []cli.Flag{
 		Destination: &ServerConfig.KubeConfigMode,
 		EnvVar:      version.ProgramUpper + "_KUBECONFIG_MODE",
 	},
+	&cli.StringFlag{
+		Name:        "helm-job-image",
+		Usage:       "(helm) Default image to use for helm controller",
```

Comment:
clarify that this is for helm jobs, not the controller. the controller that creates jobs runs in the main k3s process.
```suggestion
		Usage:       "(helm) Default image to use for helm jobs",
```